react/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC, Fragment } from 'react'
+import { generateBlockClass, BlockClass } from '@vtex/css-handles'
+import styles from './styles.css'
+
+interface Props {
+  zIndexOffset?: number
+}
+
+const StackLayout: FC<Props & BlockClass> = ({ children, blockClass, zIndexOffset = 0 }) => {
+  return (
+    <div className={`${generateBlockClass(styles.stackContainer, blockClass)} relative flex flex-grow-1 items-stretch`}>
+      <Fragment>
+        {React.Children.toArray(children).map((child, idx) => {
+          if (idx === 0) {
```

Comment:
quando vejo `idx` no codigo, nao preciso nem ver no blame pra saber q é do fidelis 😂 

(n to criticando nao, só um comentario msm hahah)